new_dawn_server/actions/api/resources.py

```diff
@@ -102,6 +127,16 @@ def obj_create(self, bundle, **kwargs):
                                          user_from_id=bundle.data.get("user_to_id"),
                                          action_type=ActionType.LIKE.value).exists():
                 self.create_match(bundle.data.get("user_from_id"), bundle.data.get("user_to_id"))
+        if bundle.data.get("action_type") == ActionType.REQUEST_TAKEN.value:
```

Comment:
We should check ACCEPT_TAKEN here since that's the moment where an ALREADY_TAKEN will be created. 

Later in line 138 we should check if the other user did REQUEST_TAKEN to the current user to validate if both sides should start ALREADY_TAKEN